extensions/amp-form/0.1/test/integration/test-integration-form.js

```diff
@@ -145,7 +147,9 @@ describes.realWin('AmpForm Integration', {
     });
   });
 
-  describeChrome.run('Submit xhr-POST', () => {
+  describeChrome.run('Submit xhr-POST', function() {
+    this.timeout(RENDER_TIMEOUT);
+
     it('should submit and render success', () => {
```

Comment:
Not sure why you moved away from arrow syntax above, but does it make sense to do so here and elsewhere in the file too?